dom/collections/HTMLCollection-own-props.html

```diff
@@ -0,0 +1,109 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>HTMLCollection getters and own properties</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<div id=log></div>
+<script>
+function append(t, tag, name) {
+  var element = document.createElement(tag);
+  if (name) {
+    element.id = name;
+  }
+  document.body.appendChild(element);
+  t.add_cleanup(function() { element.remove(); });
+  return element;
+}
+
+test(function() {
+  var name = "named", tag = "a";
```

Comment:
I like how only single-character tag names are used in alphabetical order in this test :)